ow_plexil/src/plexil-adapter/OwAdapter.cpp

```diff
@@ -337,18 +337,18 @@ static void set_light_intensity (Command* cmd, AdapterExecInterface* intf)
   args[0].getValue (side);
   args[1].getValue (intensity);
   if (side != "left" && side != "right") {
-    ROS_ERROR ("set_light_intensity: side was %s, should be 'left' or 'right'",
+    ROS_ERROR ("light_set_intensity: side was %s, should be 'left' or 'right'",
                side.c_str());
     valid_args = false;
   }
   if (intensity < 0.0 || intensity > 1.0) {
-    ROS_ERROR ("set_light_intensity: intensity was %f, "
+    ROS_ERROR ("light_set_intensity: intensity was %f, "
                "should be in range [0.0 1.0]", intensity);
     valid_args = false;
   }
   if (valid_args) {
     unique_ptr<CommandRecord>& cr = new_command_record(cmd, intf);
-    OwInterface::instance()->setLightIntensity (side, intensity, CommandId);
+    OwInterface::instance()->lightSetIntensity (side, intensity, CommandId);
     acknowledge_command_sent(*cr);
   }
```

Comment:
Good suggestion, though I'm not sure if that's better than the redundant check, which has its advantages (e.g. performance, customizing the message).  It would apply to many actions.  Will keep this in mind during future development.